object-cache.php

```diff
@@ -1191,38 +1185,37 @@ public function build_client_parameters( $redis_server ) {
 		if ( empty( $redis_server ) ) {
 			// Attempt to automatically load Pantheon's Redis config from the env.
 			if ( isset( $_SERVER['CACHE_HOST'] ) ) {
-				$redis_server = array(
-					'host'     => $_SERVER['CACHE_HOST'],
-					'port'     => $_SERVER['CACHE_PORT'],
-					'auth'     => $_SERVER['CACHE_PASSWORD'],
-					'database' => isset( $_SERVER['CACHE_DB'] ) ? $_SERVER['CACHE_DB'] : 0,
-				);
+				$redis_server = [
+					'host' => wp_strip_all_tags( $_SERVER['CACHE_HOST'] ),
+					'port' => isset( $_SERVER['CACHE_PORT'] ) ? wp_strip_all_tags( $_SERVER['CACHE_PORT'] ) : 0,
+					'auth' => isset( $_SERVER['CACHE_PASSWORD'] ) ? wp_strip_all_tags( $_SERVER['CACHE_PASSWORD'] ) : '',
```

Comment:
@jazzsequence FYI, just noting that this line is incorrect and the default value should have been `null` as I had indicated in my PR to fixed this. Also, the default port should not be `0` as that is completely invalid for Redis and should be `6379`.

https://github.com/pantheon-systems/wp-redis/pull/360

Reply:
@jazzsequence noting also that changes should have fixed the issues called out in https://github.com/pantheon-systems/wp-redis/issues/359 but in fact broke our patch which we will now have to redo.

Reply:
These have been updated in #428. Thanks for digging this up.